modules/caddyhttp/rewrite/caddyfile.go

```diff
@@ -25,20 +27,116 @@ func init() {
 
 // parseCaddyfile sets up the handler from Caddyfile tokens. Syntax:
 //
-//     rewrite [<matcher>] <to>
+//     rewrite [<matcher>] [<to>] {
+//         to                <string>
```

Comment:
I wonder if "to" should be called "uri" since that makes it more consistent with the "method" subdirective, which rewrites the method.

Reply:
That makes more sense to me too.